src/plugin/src/test/java/com/defenseunicorns/uds/keycloak/plugin/UpdateX509Test.java

```diff
@@ -122,7 +123,8 @@ public void setupMockBehavior() throws Exception {
                     return Stream.of(userModel);
                 });
 
-        CryptoIntegration.init(this.getClass().getClassLoader());
+//        CryptoIntegration.init(this.getClass().getClassLoader());
+//        CryptoIntegration.setProvider(new FIPS1402Provider());
```

Comment:
We can't reinitialize the provider as the Java Security Provider substitution happens at the JVM level. PowerMock uses separated Class Loaders to do mocks. This obviously causes errors (as the Security Providers are loaded by different Class Loaders).